src/librustc/session/config.rs

```diff
@@ -1104,6 +1104,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "enable ThinLTO when possible"),
     inline_in_all_cgus: Option<bool> = (None, parse_opt_bool, [TRACKED],
         "control whether #[inline] functions are in all cgus"),
+    pretty_json_error_format: bool = (false, parse_bool, [UNTRACKED],
```

Comment:
Is it necessary to add a new option? can `-Z unstable_options` be reused?

Reply:
oh right, that works :D